src/main/java/com/shapesecurity/salvation/directives/Directive.java

```diff
@@ -19,11 +20,23 @@
 
     @Nonnull private Set<Value> values;
 
+    public boolean isFetchDirecive() {
+        return isFetchDirecive;
+    }
+
+    private boolean isFetchDirecive = false;
```

Comment:
It's a typo. Also, you don't need this if you do `instanceof FetchDirective` instead.